plugins/fe_events_moodle/classes/event/fe_event.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace local_fe_events_moodle\event;
+
+defined('MOODLE_INTERNAL') || die();
+
+class fe_event extends \core\event\base {
+
```

Comment:
Let's go ahead and address the PHPdoc check errors from this plugin:

![image](https://user-images.githubusercontent.com/839025/180849889-2951cd07-b845-4b0d-be1d-3682fbbd8344.png)

See [this comment](https://github.com/openstax/raise-moodle-spikes/pull/40#discussion_r907451215) in another PR as far as context on these checkers and how to run them.